packages/create-styles/src/create-style-system/create-core-element.js

```diff
@@ -136,16 +135,11 @@ export const createCoreElement = (tagName, options) => {
 			}
 		}
 
-		// eslint-disable-next-line
-		const refs = React.useMemo(() => {
-			return mergeRefs([forwardedRef, ref]);
```

Comment:
I have no idea if it's necessary but another fix could be as simple as:
```js
return forwardedRef ? mergeRefs([forwardedRef, ref]) : ref;
```
Maybe it would resolve the issue with Vercel?